package.json

```diff
@@ -34,6 +34,7 @@
     "@babel/core": "7.22.9",
     "@babel/eslint-parser": "7.22.9",
     "@babel/preset-env": "7.22.9",
+    "@nightwatch/vrt": "^3.0.0",
```

Comment:
Could this be in the `optionalDependencies` list, where the `nightwatch` dependency resides? That would keep the dependencies lighter for the unit tests GitHub action, and a normal `npm install` command would still install it

Reply:
Hm, I added it to `devDependencies` based on the installation instructions. Test framework dependencies are a commonly cited example of when to use dev dependencies. IIUC, optionalDependencies are for dependencies that can be used by app code but are not strictly required.

https://docs.npmjs.com/cli/v9/configuring-npm/package-json

Our devDependencies list includes many other test packages like chai, karma, etc. I'm not sure why nightwatch and others would be in optionalDependences.

Reply:
Got it, thanks. Updated.

Reply:
Thanks! Just noticed the `check` omits the optional deps too https://github.com/subscriptions-project/swg-js/blob/65ccb7d85cd8a0080fe90808a1941de3cf083263/.github/workflows/ci.yaml#L22